Source/WebCore/css/CSSSelector.cpp

```diff
@@ -318,7 +318,7 @@ PseudoId CSSSelector::pseudoId(PseudoElement type)
 
 std::optional<CSSSelector::PseudoElement> CSSSelector::parsePseudoElement(StringView name, const CSSSelectorParserContext& context)
 {
-    if (name.isNull())
+    if (name.isEmpty())
         return std::nullopt;
 
     auto type = parsePseudoElementString(name);
```

Comment:
Probably a separate PR, but I kinda wish this was named `parsePseudoElementWithoutContext` or something.